module-services/service-cellular/DTMFCode.hpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+#include <stdint.h>
+#include <string>
+
+class DTMFCode
+{
+  public:
+    explicit DTMFCode(char digitChar);
+    explicit DTMFCode(const char *digitString);
+    [[nodiscard]] auto getDigit() -> char;
+    [[nodiscard]] auto getDigitASCIICode() -> char;
+    [[nodiscard]] auto getDTMFCodeString() -> const std::string;
```

Comment:
why we have such redundancy: we are using DTMF so we know that we are using DTMF string if we return string? So why not even just add `operator str()` or if you really have to `toString` method?

I strongly believe that adding `DTMFCode`in `DTMFCode` class doesn't carry any information :)

Reply:
added string cast operator